source/macos/AdaptiveCards/AdaptiveCardsTests/Views/ACRChoiceSetViewTests.swift

```diff
@@ -0,0 +1,33 @@
+@testable import AdaptiveCards
+import AdaptiveCards_bridge
+import XCTest
+
+class ACRChoiceSetViewtests: XCTestCase {
+    private var choiceSetView: ACRChoiceSetView!
+    private var renderConfig: RenderConfig!
+    
+    override func setUp() {
+        super.setUp()
+        renderConfig = RenderConfig(isDarkMode: false, buttonConfig: .default, supportsSchemeV1_3: false, hyperlinkColorConfig: .default, inputFieldConfig: .default,checkBoxButtonConfig: nil, radioButtonConfig: nil)
+        choiceSetView = ACRChoiceSetView(renderConfig: renderConfig)
+    }
+    
+    func testRadioButtonClickAction() {
+        choiceSetView.isRadioGroup = true
+        let button1 = choiceSetView.setupButton(attributedString: .init(string: "1"), value: "Button1")
+        let button2 = choiceSetView.setupButton(attributedString: .init(string: "2"), value: "Button2")
+        // The clicking of the button is handled in ACRChoiceButton and ACRChoiceSetView parallely, with ACRChoiceSetView only making sure the previous radio button is turned off when another radio button is pressed
```

Comment:
```suggestion
        let button1 = choiceSetView.setupButton(attributedString: NSAttributedString(string: "1"), value: "Button1")
        let button2 = choiceSetView.setupButton(attributedString: NSAttributedString(string: "2"), value: "Button2")
        // The clicking of the button is handled in ACRChoiceButton and ACRChoiceSetView parallely, with ACRChoiceSetView only making sure the previous radio button is turned off when another radio button is pressed
```

Nit: as a practice, We can avoid using `.init` and use the instance name.